### DISCUSSION TITLE
List of icons available in chosen icon-set?

### DISCUSSION THREAD

**Discussion by danielwillms** [quasarframework/quasar#15302] [General - Components / Directives / etc]
Hey!
I wanted to build my own icon-picker but can't seem to find if there's a list/array of all the icon-names available (in this case for 'Material Symbols Sharp').
The only files I found were 
- the `icons.json` at `@quasar/extras/material-symbols-sharp`, but the names are camelCased and not snake_cased, like you need for the QIcon name-prop
- `material-icons-sharp.umd.js` in the QIconPicker extension at `@quasar/quasar-ui-qiconpicker/dist/icon-set`, but as the name implies it's for Material **Icons**, not Material **Symbols**.

Am I overseeing the obvious here or is this not available?

Thanks in advance,
Daniel

**Comment by MilosPaunovic** [comment]
Hi Daniel,

Take a look at this handy project our own Jeff created to help users explore all of the available icon packs for Quasar:

https://iconexplorer.app

**Reply by danielwillms** [reply]
Hi Milos,

thank you for your answer!
I've already stumbled upon the Iconexplorer, but unfortunately it's not what I'm looking for.

What I'm looking for is basically a package/json/js file where I can get an array of all the available Icons in form of Strings, which are compatible with the QIcon-Component.
For example 
`["sym_s_home", "sym_s_mood", "sym_s_arrow_up", ...]`

**Reply by MilosPaunovic** [reply]
Take a dive into the `extras` folder of this repository.
